src/cli/run.ts

```diff
@@ -134,7 +134,7 @@ export const runTask = async (
       break;
 
     case 'help':
-      taskHelp(config, config.logger, sys);
+      await taskHelp(config, config.logger, sys);
```

Comment:
This `await` is not strictly necessary for this bug fix, but was added for consistency.